recipes-mender/mender/files/mender.service

```diff
@@ -5,7 +5,7 @@ Description=Mender OTA update service
 Type=simple
 User=root
 Group=root
-ExecStart=/usr/bin/mender -daemon
+ExecStart=/usr/bin/mender -daemon -log-level=debug -log-file=/var/log/mender.log
```

Comment:
I think we should not enable this generally. What do you need this for? Testing? You can just enable it in your local build.

Otherwise I agree with using central log files.